packages/kit/src/core/sync/write_client_manifest.js

```diff
@@ -93,5 +96,6 @@ export function write_client_manifest(manifest_data, output) {
  * @param {number} id
  */
 function get_node_id(nodes, id) {
+	console.log(nodes[id]);
```

Comment:
```suggestion
```

I also don't see this "second commit" you mention in the PR description.

Reply:
oops forgot to push it - done